src/backend/core/api/serializers.py

```diff
@@ -131,15 +133,20 @@ class TeamSerializer(serializers.ModelSerializer):
 
     abilities = serializers.SerializerMethodField(read_only=True)
     accesses = TeamAccessSerializer(many=True, read_only=True)
+    slug = serializers.SerializerMethodField()
 
     class Meta:
         model = models.Team
-        fields = ["id", "name", "accesses", "abilities"]
-        read_only_fields = ["id", "accesses", "abilities"]
+        fields = ["id", "name", "accesses", "abilities", "slug"]
+        read_only_fields = ["id", "accesses", "abilities", "slug"]
 
     def get_abilities(self, team) -> dict:
         """Return abilities of the logged-in user on the instance."""
         request = self.context.get("request")
         if request:
             return team.get_abilities(request.user)
         return {}
+
+    def get_slug(self, instance):
+        """Return slug from the team's name."""
+        return slugify(instance.name)
```

Comment:
ditto
```suggestion
```

Reply:
see answer on slug field, l.136.

Reply:
Ok in this case I propose to call a common method on the model to have unicity of the slug computation:
```
instance.compute_slug
```
and on the model:
```
def get_slug(self):
    return self.slug or slugify(self.name)
```